javascript/js-declarations/src/org/jetbrains/dukat/js/declarations/toplevel/JSClassDeclaration.kt

```diff
@@ -0,0 +1,6 @@
+package org.jetbrains.dukat.js.declarations.toplevel
+
+data class JSClassDeclaration(
+        override val name: String,
+        val methods: MutableSet<JSFunctionDeclaration>
```

Comment:
let's name it members since we can have in theory not only methods

Reply:
I've also said this in the comment regarding the topLevelDeclarations in the JSModuleDeclaration:

> Since the JSModuleDeclaration and the JSClassDeclaration both contain values at a top level, that need to be treated in a simmilar way, I've since had them both implement an interface "JSScopedDeclaration", which has a "val scopeDeclarations: MutableMap<String, JSDeclaration>". This replaces the topLevelDeclarations in JSModuleDeclaration, and the methods in JSClassDeclaration.